src/main/java/org/jenkinsci/plugins/workflow/support/pickles/ExecutorPickle.java

```diff
@@ -122,15 +122,10 @@ protected Executor tryResolve() throws Exception {
                         // Thus we're guaranteeing the execution began and the Node is now unknown.
                         // Theoretically it's safe to simply fail earlier when rehydrating any EphemeralNode... but we're being extra safe.
                         if (placeholder.getCookie() != null && Jenkins.getActiveInstance().getNode(placeholder.getAssignedLabel().getName()) == null ) {
-                            if (isEphemeral() || System.nanoTime() > endTimeNanos) {
+                            if (System.nanoTime() > endTimeNanos) {
                                 Queue.getInstance().cancel(item);
-                                if (isEphemeral()) {
```

Comment:
Then you might as well delete the `ephemeral` flag.

Reply:
I'd like to keep that info handy for future use, plus it avoids grumping from XStream because the field no longer exists.